packages/web-components/src/components/leaving-ibm/leaving-ibm-modal-heading.ts

```diff
@@ -21,6 +21,13 @@ const { stablePrefix: ddsPrefix } = ddsSettings;
  */
 @customElement(`${ddsPrefix}-leaving-ibm-modal-heading`)
 class DDSLeavingIbmModalHeading extends BXModalHeading {
+  connectedCallback() {
+    if (!this.hasAttribute('id')) {
+      this.setAttribute('id', 'leaving-ibm-heading');
+    }
+    super.connectedCallback();
+  }
+
```

Comment:
To avoid (theoretical) ID duplication (though it's less likely for an app to have more than one leaving IBM modal), we can let `<dds-leaving-ibm-modal>` auto-generate the ID:

```suggestion
```